lib/fizzy/instantiate.hpp

```diff
@@ -46,9 +46,6 @@ class ExecuteFunction
     std::any m_host_context;
 
 public:
-    /// Default function constructor.
-    ExecuteFunction() noexcept = default;
```

Comment:
Was the constructor of the line `fizzy::ImportedFunction imported_func;` using this?